spec/spec_helper.rb

```diff
@@ -1,6 +1,11 @@
 $LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..', 'lib'))
 $LOAD_PATH.unshift(File.dirname(__FILE__))
 
+require 'simplecov'
+SimpleCov.start 'rails' do
```

Comment:
I don't know much about simplecov, but what's up with Rails here?

Reply:
It is the most suitable preset. I have already restricted it.
https://github.com/colszowka/simplecov/blob/master/lib/simplecov/profiles/rails.rb